src/nucomp.h

```diff
@@ -193,7 +193,7 @@ void qfb_nucomp(qfb_t r, const qfb_t f, const qfb_t g, mpz_t& D, mpz_t& L)
 
 // a = b * c
 void nucomp_form(form &a, form &b, form &c, integer &D, integer &L) {
-    qfb fr, fr2, fr3;
```

Comment:
actually, it it occurred to me this morning; do we ever want to declare `qfb` objects that are uninitialized? If not, we should just do this in its constructor and rest assured we won't have this problem again.